pkg/network/node/pod.go

```diff
@@ -67,7 +67,8 @@ type podManager struct {
 
 	// Things only accessed through the processCNIRequests() goroutine
 	// and thus can be set from Start()
-	ipamConfig []byte
+	ipamConfig   []byte
+	reattachPods map[string]*corev1.Pod
```

Comment:
We should be consistent about the naming; if it's just for reattaching, then `addPodsToCache` / `delPodsFromCache` should reflect that too. OTOH if it's not just for reattaching then this field should have a more generic name.

Making it only for reattaching might make sense... In that case there's no need for `delPodsFromCache` to take an argument, since the only allowed call would be to clear the whole cache. So it could be like `setReattachPodsCache(pods)` / `clearReattachPodsCache()`. Or `startReattachPods(pods)` / `endReattachPods()` or something